meta/documents/changelog_en.md

```diff
@@ -1,5 +1,12 @@
 # Release Notes for IO
 
+## v4.3.1 (2019-10-01) <a href="https://github.com/plentymarkets/plugin-io/compare/4.3.0...4.3.1" target="_blank" rel="noopener"><b>Overview of all changes</b></a>
+
+### Fixed 
+
+- During analyse the url, the current language was not correct determined. This has been fixed.
```

Comment:
Due to an error, language-specific URLs could not be determined correctly. This has been fixed.